psi4/src/psi4/libmints/multipolepotential.cc

```diff
@@ -0,0 +1,221 @@
+/*
+ * @BEGIN LICENSE
+ *
+ * Psi4: an open-source quantum chemistry software package
+ *
+ * Copyright (c) 2007-2019 The Psi4 Developers.
+ *
+ * The copyrights for code used from other parties are included in
+ * the corresponding files.
+ *
+ * This file is part of Psi4.
+ *
+ * Psi4 is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * Psi4 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with Psi4; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * @END LICENSE
+ */
+
+#include "psi4/libmints/multipolepotential.h"
+#include "psi4/libmints/basisset.h"
+#include "psi4/libmints/integral.h"
+
+#define MAX(a, b) ((a) > (b) ? (a) : (b))
```

Comment:
Can you use `std::max`?

Reply:
Don't even know where `MAX` is used... it's defined in all the *.cc files involving integrals.

`ack "define MAX"`:
```
dipole.cc
38:#define MAX(a, b) ((a) > (b) ? (a) : (b))

efpmultipolepotential.cc
33:#define MAX(a, b) ((a) > (b) ? (a) : (b))

rel_potential.cc
39:#define MAX(a, b) ((a) > (b) ? (a) : (b))

kinetic.cc
36:#define MAX(a, b) ((a) > (b) ? (a) : (b))

tracelessquadrupole.cc
35:#define MAX(a, b) ((a) > (b) ? (a) : (b))

pseudospectral.cc
34:#define MAX(a, b) ((a) > (b) ? (a) : (b))

potential.cc
40:#define MAX(a, b) ((a) > (b) ? (a) : (b))

overlap.cc
36:#define MAX(a, b) ((a) > (b) ? (a) : (b))

eribase.cc
43:#define MAX(a, b) ((a) > (b) ? (a) : (b))

quadrupole.cc
35:#define MAX(a, b) ((a) > (b) ? (a) : (b))

electricfield.cc
36:#define MAX(a, b) ((a) > (b) ? (a) : (b))

matrix.cc
74:#define MAX(x, y) ((x) > (y) ? (x) : (y))

multipolepotential.cc
33:#define MAX(a, b) ((a) > (b) ? (a) : (b))

electrostatic.cc
38:#define MAX(a, b) ((a) > (b) ? (a) : (b))
```

Reply:
yeah, please delete macro and use `std::max` in any new files. we'll clean up the existing ones later.

Reply:
Done.